synfig-studio/src/gui/resourcehelper.cpp

```diff
@@ -74,28 +71,24 @@ synfig::String studio::ResourceHelper::get_image_path(const synfig::String& imag
 
 synfig::String studio::ResourceHelper::get_ui_path()
 {
-#ifdef _WIN32
-# ifdef UI_DIR
-#  undef UI_DIR
-#  define UI_DIR "share/synfig/ui"
-# endif
-#endif
-
 #ifndef UI_DIR
-# define UI_DIR "/usr/local/share/synfig/ui"
+#   define UI_DIR "/usr/local/share/synfig/ui"
 #endif
 
 	std::string uipath;
-#ifdef _WIN32
-	uipath=App::get_base_path()+'/'+UI_DIR;
-#else
-	uipath=UI_DIR;
-#endif
+	uipath=uipath=App::get_base_path()+"/share/synfig/ui";
 	char* synfig_root=getenv("SYNFIG_ROOT");
 	if(synfig_root) {
 		uipath=synfig_root;
```

Comment:
make it one line

Reply:
It can't be since operation on `synfig_root (char *)` and `"what/ever/path (char[n])` is invalid. Same for the one which is not merged into one line in `get_image_path()`.

Reply:
Ok. You can use:
```
ui_path = std::string(synfig_root) + "/share/synfig/ui";
```
This will be the same.